src/App.tsx

```diff
@@ -1,11 +1,11 @@
+import React from "react";
+import Landing from "@/pages/landing/LandingContainer.js";
```

Comment:
Maybe you could improve this a little bit to import just like `import Landing from "@/pages";`

Reply:
I like it, I've added a new absolute path for @landing, and one for @main to use in the future